tasks/main.yml

```diff
@@ -1,26 +1,32 @@
 ---
 
-- include: configure_acng_server.yml
-  when: (apt is defined and apt) and (apt != True and apt == ansible_fqdn)
+#- acng Client
 
 - include: configure_acng_client.yml
-  when: (apt is defined and apt) and apt != True
+  when: apt != True
+
+#- acng Server
+
+- include: configure_acng_server.yml
```

Comment:
You should configure acng server before acng client, currently client is configured before server and after that cache server cannot be installed because cache server is not installed.